addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -209,11 +212,4 @@ def _compute_l10n_latam_available_document_types(self):
     def _compute_l10n_latam_document_type(self):
         for rec in self.filtered(lambda x: x.state == 'draft'):
             document_types = rec.l10n_latam_available_document_type_ids._origin
-            invoice_type = rec.move_type
-            if invoice_type in ['out_refund', 'in_refund']:
-                document_types = document_types.filtered(lambda x: x.internal_type not in ['debit_note', 'invoice'])
-            elif invoice_type in ['out_invoice', 'in_invoice']:
-                document_types = document_types.filtered(lambda x: x.internal_type not in ['credit_note'])
-            if rec.debit_origin_id:
-                document_types = document_types.filtered(lambda x: x.internal_type == 'debit_note')
```

Comment:
This changes the behavior, are you sure a localization doesn't rely on this?

Reply:
@jjscarafia whenever you can, remember to add your comments about this point.
You shared this link during the call related to this point: https://github.com/odoo/odoo/blob/f5c0552a2b2e975db8745219af044638449c3312/addons/l10n_ar/models/account_move.py#L90

Also, I think you said this was breaking the AR debit note creation. But I can't recall the other comment you had.
Thanks!